packages/lib-utils/src/types/k8s.ts

```diff
@@ -58,3 +58,13 @@ export type Patch = {
   path: string;
   value?: unknown;
 };
+
+export type K8sVerb =
+  | 'create'
+  | 'get'
+  | 'list'
+  | 'update'
+  | 'patch'
+  | 'delete'
+  | 'deletecollection'
+  | 'watch';
```

Comment:
```suggestion
```
This is in [core.ts](https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/lib-core/src/types/core.ts).

Reply:
@andrewballantyne that type is not exported from the core package. I can update us to use it in the util package, but will also need to change the core package.

Reply:
@andrewballantyne I do not see that type being exported in the `dist` folder for the `core` package. I can pull it in directly from the src folder or export it from the entrypoint?

Reply:
This should be easily fixed by exporting it normally out the index (or whatever we call the file we will create to handle the split) 🤔 

Not sure why it would be a problem. @vojtechszocs ?